packages/vega-parser/src/parsers/guides/guide-util.js

```diff
@@ -86,3 +87,80 @@ export function extendOffset(value, offset) {
     : !isObject(value) ? { value, offset }
     : { ...value, offset: extendOffset(value.offset, offset) };
 }
+
+const DISCRETE_SCALES = new Set(['ordinal', 'band', 'point']);
+
+function formatType(_, scaleType) {
+  const formatType = _('formatType');
+
+  if (formatType) {
+    return formatType;
+  }
+  if (scaleType === Time) {
+    return 'time'
```

Comment:
Time === 'time' and UTC === 'utc', no?

Reply:
Also, these lines are missing trailing semi-colons

Reply:
Yes. I like writing

```
if (x === 1) {
	return true;
}
```

instead of

```
return x === 1;
```

Reply:
Simplified to 

```
function formatType(_, scaleType) {
  return _('formatType') || scaleType;
}
```

I think there is value in having this in a function.